data/core/common_data.sh

```diff
@@ -61,6 +61,20 @@ initialize_package_manager_pkg() {
   PACKAGE_MANAGER_AUTOCLEAN="pkg -y autoclean"
 }
 
+initialize_package_manager_pacman() {
+  DEFAULT_PACKAGE_MANAGER="pacman"
+  PACKAGE_MANAGER_INSTALL="pacman -S"
+  PACKAGE_MANAGER_FIXBROKEN="pacman -Sy"
+  PACKAGE_MANAGER_UNINSTALL="pacman -Ry"
+  PACKAGE_MANAGER_UPDATE="pacman -Sy"
+  PACKAGE_MANAGER_UPGRADE="pacman -Syu"
+  PACKAGE_MANAGER_INSTALLPACKAGE=":"
+  PACKAGE_MANAGER_INSTALLPACKAGES=":"
```

Comment:
Recursive install of packages using the low-level package manager. At least provide a command to install all packages recursively, aka all packages in a folder.